asciidoctor-tests/TestAdocLinks.lua

```diff
@@ -0,0 +1,125 @@
+-- TestAdocLinks.lua - a test to verify that all external links in adoc files are functional
+-- Copyright (C) 2014-2016 Zach Rhoads
+
+-- This program is free software:  you can redistribute it and/or modify it
+-- under the terms of  the  GNU General Public License  as published by the
+-- Free Software Foundation, version 3 of the License.
+--
+-- This program  is  distributed  in the hope  that it will be useful,  but
+-- WITHOUT  ANY WARRANTY;  without  even the implied warranty of MERCHANTA-
+-- BILITY or  FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+-- License for more details.
+--
+-- You should have received a copy of the GNU General Public License  along
+-- with this program. If not, see <http://www.gnu.org/licenses/>.
+
+TestAdocLinks = {
+	metadata = {
+		description = "Verify that all external links in adoc files are functional",
+		authors = "Zach Rhoads",
+		changed = "2016-07-15",
+		tags = {"AsciiDoc"}
+	},
+	requires = {curl},
```

Comment:
This should be:

    requires = {"curl"}

Without the double quotes, Emender does not properly check the availability of the tool in the system.